### PR TITLE
update readme with example for ignoring multiple directories

### PR DESCRIPTION
**Pull request by jasonhinkle**
Add example for ignoring multiple folders

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ This will:
 
 You should be able to launch the app on the platform you built for. If not, check your settings and try again.
 
-**Be careful** not to include node_modules you don't want into your final app. `Electron-packager`, `electron-prebuilt` and `.git` will be ignored by default. You can use `--ignore` to ignore files and folders, e.g. `--ignore=node_modules/electron-packager`.
+**Be careful** not to include node_modules you don't want into your final app. `Electron-packager`, `electron-prebuilt` and `.git` will be ignored by default. You can use `--ignore` to ignore files and folders, e.g. `--ignore=node_modules/electron-packager` or `--ignore="(node_modules/atom-shell|node_modules/node-gyp)"`
```
Comment (sindresorhus) [inline readme.md]: `atom-shell` is no more, so find a better example module.

And can be simplified to:

`--ignore="node_modules/(atom-shell|node-gyp)"`

**Comment by sindresorhus** [comment]
Give the pull request and commit a proper title.

**Comment by jasonhinkle** [comment]
made the suggested updates.  hope this might help someone as it took me a while to figure out the syntax.

File: readme.md [hunk 2]
```diff
@@ -49,9 +49,9 @@ This will:
 - Find or download the correct release of Electron
 - Use that version of electron to create a app in `cwd` named using `appname` for the platform you specified
 
-You should be able to launch the app on the platform you built for. If not, check your settings and try again.
+You should be able to launch the app on the platform you built for. If not, check your settings and try again.
```
Comment (sindresorhus) [inline readme.md]: don't introduce whitespace

File: readme.md [hunk 3]
```diff
@@ -49,9 +49,9 @@ This will:
 - Find or download the correct release of Electron
 - Use that version of electron to create a app in `cwd` named using `appname` for the platform you specified
 
-You should be able to launch the app on the platform you built for. If not, check your settings and try again.
+You should be able to launch the app on the platform you built for. If not, check your settings and try again. 
 
-**Be careful** not to include node_modules you don't want into your final app. `Electron-packager`, `electron-prebuilt` and `.git` will be ignored by default. You can use `--ignore` to ignore files and folders, e.g. `--ignore=node_modules/electron-packager`.
+**Be careful** not to include node_modules you don't want into your final app. `Electron-packager`, `electron-prebuilt` and `.git` will be ignored by default. You can use `--ignore` to ignore files and folders, e.g. `--ignore=node_modules/electron-packager` or `--ignore="node_modules/(electron-packager|electron-prebuilt)"`
```
Comment (sindresorhus) [inline readme.md]: should be a `.` at the end.

**Comment by jden** [comment]
Looks good to me! Could you please rebase and squash your commits & then it can be merged.

**Comment by jasonhinkle** [comment]
ok I think its ready to go

**Comment by jden** [comment]
Thanks!

**Pull request merged**